automaxlair/da_controller.py

```diff
@@ -45,13 +45,13 @@ def __init__(
         self.window_name = 'AutoMaxLair Output'
 
         # Read values from the config.
-        self.boss = config['default']['BOSS'].lower().replace(' ', '-')
+        self.boss = config['BOSS'].lower().replace(' ', '-')
 
-        self.base_ball = config['default']['BASE_BALL']
-        self.base_balls = int(config['default']['BASE_BALLS'])
-        self.legendary_ball = config['default']['LEGENDARY_BALL']
-        self.legendary_balls = int(config['default']['LEGENDARY_BALLS'])
-        self.mode = config['default']['MODE'].lower()
+        self.base_ball = config['BASE_BALL']
+        self.base_balls = int(config['BASE_BALLS'])
```

Comment:
Do we still need to convert to int or will the value be read as an integer automatically? Or are you rounding in case someone enters a float?

Reply:
I'll remove the int call, as long as there isn't a decimal it'll be an int